service/src/event/service.rs

```diff
@@ -237,6 +238,126 @@ impl CeramicEventService {
         })
     }
 
+    #[allow(dead_code)]
+    async fn transform_raw_events_to_conclusion_events(
+        &self,
+        raw_event: ceramic_event::unvalidated::Event<Ipld>,
+        delivered: i64,
+    ) -> Result<ConclusionEvent> {
+        let stream_cid = raw_event.id();
+        let init_event = self.get_event_by_cid(stream_cid).await?;
+        let init = ConclusionInit::try_from(init_event).unwrap();
+        let event_cid = *raw_event.id();
```

Comment:
This is incorrect, the event_cid is not the stream_cid (unless its an init event). Does the raw_event know its own cid? That is what we need to use. Maybe we need a new method on unvalidated::Event to get the cid of itself.

Reply:
we can also pass it as a parameter into this method